### PR TITLE
tape: `skip` property can be a string

### PR DESCRIPTION
**Pull request by ljharb**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/tape/tape-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5dfdabcccaffede57b97e6d230fca609dfbbd1b5
</p>

**Comment by ljharb** [comment]
Blocked on https://github.com/microsoft/DefinitelyTyped-tools/pull/1113

File: types/tape/package.json [hunk 1]
```diff
@@ -2,12 +2,19 @@
     "private": true,
     "name": "@types/tape",
     "version": "5.6.9999",
+    "exports": {
+		".": "./index.js",
+		"./lib/default_stream": "./lib/default_stream.js",
+		"./lib/results": "./lib/results.js",
+		"./lib/test": "./lib/test.js",
```
Comment (jakebailey) [inline types/tape/package.json:9]: Do these files exist on DT? I don't see them, so I assume this will break?

Comment (ljharb) [reply types/tape/package.json:9]: oh, hm - no, they don't - i just copied the exports field verbatim. should these be pointing to the d.ts files instead?

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/tape/package.json
1. types/tape/tape-tests.ts
1. types/tape/lib/test.d.ts
1. types/tape/lib/results.d.ts
1. types/tape/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 55e8642c766a5b82d3d241f5ddb6660bcf422294
</p>

File: types/tape/package.json (hunk 1 above)
Comment (jakebailey) [reply types/tape/package.json]: Copying verbatim is fine, but it was more about the other `./lib/*` files seemingly not existing on DT.

**Review by jakebailey — APPROVED** [review]

**Comment by ljharb** [comment]
Ready to merge

**Comment by captain-igloo** [comment]
I am getting an error after upgrading to @types/tape 5.6.5:

```
node_modules/@ljharb/through/through.d.ts:3:8 - error TS1259: Module '"stream"' can only be default-imported using the 'esModuleInterop' flag

3 import stream from 'stream';
         ~~~~~~

  node_modules/@types/node/stream.d.ts:1721:5
    1721     export = internal;
             ~~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.


Found 1 error in node_modules/@ljharb/through/through.d.ts:3
```

This is easily reproducable:

```
git clone git@github.com:captain-igloo/tape-bug.git
cd tape-bug
npm ci
npm run build
```

I don't want to add esModuleInterop in my project.

**Comment by ljharb** [comment]
@captain-igloo that would be something you need to file on https://github.com/ljharb/through, but I'd just tell you that TS's module system is utterly broken without esModuleInterop and synthetic imports, and that you should just enable it.

**Comment by jakebailey** [comment]
Your example also doesn't configure any options (e.g. module/moduleResolution), so you're getting the worst possible settings for everything

**Comment by captain-igloo** [comment]
ok thanks

**Pull request merged**